src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -37,9 +37,19 @@ public void ConfigureServices(IServiceCollection services)
                 // HACK We need to disable EndpointRouting in order to run properly in 3.1
                 System.Reflection.PropertyInfo prop = options.GetType().GetProperty("EnableEndpointRouting");
                 prop?.SetValue(options, false);
-
             }).SetCompatibilityVersion(CompatibilityVersion.Latest);
 
+            services.Configure<ApiBehaviorOptions>(options =>
+            {
+                options.InvalidModelStateResponseFactory = context =>
+                {
+                    var details = new ValidationProblemDetails(context.ModelState);
+                    var result = new BadRequestObjectResult(details);
+                    result.ContentTypes.Add("application/problem+json");
```

Comment:
Doesn't have to be part of this checkin, but we should make sure that our other error results also use this media type, if possible.